### PR TITLE
LIBCLANG_PATH hook [updated]

### PR DESCRIPTION
**Pull request by JJJollyjim**
###### Motivation for this change

This is #67725 rebased against current staging, posting it as a separate pull request to trigger an ofborg eval.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/version-management/pijul/default.nix [hunk 1]
```diff
@@ -40,6 +40,7 @@ in rustPlatform.buildRustPackage rec {
   '';
 
   nativeBuildInputs = [ pkgconfig clang ];
+  bulidInputs = [ libclang ];
```
Comment (dreverri) [inline pkgs/applications/version-management/pijul/default.nix:43]: Should this be `buildInputs`?

**Comment by bjornfor** [comment]
What's the status? Was it just forgotten?

**Comment by kvark** [comment]
@JJJollyjim could you resolve merge conflicts? The fix is very much needed, still.

**Comment by winterqt** [comment]
This was added (via an opt-in setup hook) in #159899, closing.

**Pull request closed without merging**